src/Symfony/Component/Translation/Translator.php

```diff
@@ -466,11 +466,12 @@ private function getFallbackContent(MessageCatalogue $catalogue)
     private function getCatalogueCachePath($locale)
     {
         $catalogueHash = sha1(serialize(array(
+            'locale' => $this->locale,
```

Comment:
I'm not sure about it , if someone wants to manually refresh the cache for `en` catalogue for example ?